transforms/language/text_encoder/python/test/test_text_encoder_python.py

```diff
@@ -0,0 +1,43 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import os
+
+from data_processing.runtime.pure_python import PythonTransformLauncher
+from data_processing.test_support.launch.transform_test import (
+    AbstractTransformLauncherTest,
+)
+from text_encoder_transform_python import TextEncoderPythonTransformConfiguration
+
+
+class TestPythonTextEncoderTransform(AbstractTransformLauncherTest):
+    """
+    Extends the super-class to define the test data for the tests defined there.
+    The name of this class MUST begin with the word Test so that pytest recognizes it as a test class.
+    """
+
+    def get_test_transform_fixtures(self) -> list[tuple]:
+        basedir = "../test-data"
+        basedir = os.path.abspath(os.path.join(os.path.dirname(__file__), basedir))
+        fixtures = []
+        launcher = PythonTransformLauncher(TextEncoderPythonTransformConfiguration())
+        fixtures.append(
+            (
+                launcher,
+                {},
+                basedir + "/input",
+                basedir + "/expected",
+                # this is added as a fixture to remove these columns from comparison
+                ["embeddings"],
```

Comment:
Are the embedding supposed to be reproducible (modulo floating point precision)?  If so, how do we deal with this - if checked in as is, we should have an issue to go with it.  I think the right solution may be to truncate floating point values in the embeddings vector to some level or reasonable precision that will make the tests pass.

Reply:
yes, we need an issue for being to address this at the core of the DPK, it cannot be the responsibility of each transform to implement such a code. (but it is its responsibility to know _when_ it could be used)